Week2/js-exercises/oddOnesOut.js

```diff
@@ -0,0 +1,24 @@
+
+const myNumbers = [1, 2, 3, 4];
```

Comment:
This array is not used anywhere. Please remove any unused code before submitting